.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+on:
+  pull_request: { }
+  push:
+    branches: [ develop, master ]
+jobs:
+  test:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.13'
```

Comment:
1.13 is pretty old now, though I suppose it's best practice to update the go version in a separate PR.